src/test/java/htsjdk/samtools/util/IntervalListWriterTest.java

```diff
@@ -26,7 +32,18 @@ void setup() {
     public void testEndToEnd() throws IOException {
         final File tempFile = File.createTempFile("IntervalListWriterTest.", ".interval_list");
         tempFile.deleteOnExit();
+        testEndToEnd(tempFile.toPath());
+    }
+
+    @Test
+    public void testEndToEndOnPath() throws IOException {
+        try(FileSystem jimfs = Jimfs.newFileSystem(Configuration.unix())) {
```

Comment:
what does `Configuration.unix()` do for us? can you dd a comment to explain this test?

Reply:
It makes the file system behave like a unix one vs a windows/ mac one.   Added a comment.